Dynamic/Types/ChoiceTrait.php

```diff
@@ -39,6 +39,8 @@ private function getChoiceOptions(
         FormFieldTranslation $translation,
         array $options
     ): array {
+        $options['placeholder'] = false;
```

Comment:
if no empty value should be displayed the content manager need to set the value to `required`. Will remove this change.